src/pages/Home/frames/ProgressToday/components/ButtonRedirect.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+
+import styled, { css } from "styled-components";
+import { ArrowLinkIcon } from "../../../../../components/Icons/ArrowLinkIcon";
+import { IThemeProps } from "../../../../../styles/theme";
```

Comment:
Ao importar tipos e sempre importante colocar um type na frente do importe
```suggestion
import type { IThemeProps } from "../../../../../styles/theme";
```

Isso e importante pois na hora de compilar ele não vai fazer a importação do tipo nesse arquivo.